omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -130,11 +130,12 @@
   opscode-expander
   bookshelf
   opscode-erchef
+  nginx
+  rabbitmq
   bootstrap
   opscode-chef-mover
   redis_lb
-  nginx
-  rabbitmq
+  elasticsearch
```

Comment:
We might want to remove this from this PR, but I also think it is fine since we default to disabled at the moment.

Reply:
Good catch. Removed!